README.md

```diff
@@ -52,6 +52,7 @@ See <https://github.com/ansible/ansible/issues/71528> for more information.
 | zookeeper_servers                        | zookeeper-nodes                                                   | See below                                                      |
 | zookeeper_servers_use_inventory_hostname | false                                                             | See below                                                      |
 | zookeeper_environment                    | "JVMFLAGS": "-javaagent:/opt/jolokia/jolokia-jvm-1.6.0-agent.jar" ||
+| zookeeper_cfg_params                     | "standaloneEnabled": "false"                                      | A key-value dictionary that will be templated into zoo.cfg     |
```

Comment:
There's no actual default for this so this could be moved into the comments column in this table for that item as an example config param.

Reply:
Good point, I introduced the comments/default columns but didn't end up using it correctly!